--- configure.ac	2014-10-08 17:48:22.654942849 +0400
+++ configure.ac.patched	2014-10-08 17:48:44.218943175 +0400
@@ -1,4 +1,4 @@
-AC_PREREQ([2.65])
+AC_PREREQ([2.63])
 AC_INIT([libsodium],[1.0.0],
   [https://github.com/jedisct1/libsodium/issues],
   [libsodium],
@@ -110,23 +110,23 @@
   ])
 ])
 
-AC_ARG_ENABLE(debug,
-[AS_HELP_STRING(--enable-debug,For maintainers only - please do not use)],
-[
-  AS_IF([test "x$enableval" = "xyes"], [
-    AS_IF([test "x$LX_CFLAGS" = "xNONE"], [
-      nxflags=""
-      for flag in `echo $CFLAGS`; do
-        AS_CASE([$flag],
-          [-O*], [ ],
-          [-g*], [ ],
-          [*], [AS_VAR_APPEND([nxflags], [" $flag"])])
-      done
-      CFLAGS="$nxflags -O0 -g3"
-    ])
-    CPPFLAGS="$CPPFLAGS -DDEBUG=1"
-  ])
-])
+#AC_ARG_ENABLE(debug,
+#[AS_HELP_STRING(--enable-debug,For maintainers only - please do not use)],
+#[
+#  AS_IF([test "x$enableval" = "xyes"], [
+#    AS_IF([test "x$LX_CFLAGS" = "xNONE"], [
+#      nxflags=""
+#      for flag in `echo $CFLAGS`; do
+#        AS_CASE([$flag],
+#          [-O*], [ ],
+#          [-g*], [ ],
+#          [*], [AS_VAR_APPEND([nxflags], [" $flag"])])
+#      done
+#      CFLAGS="$nxflags -O0 -g3"
+#    ])
+#    CPPFLAGS="$CPPFLAGS -DDEBUG=1"
+#  ])
+#])
 
 AC_SUBST([MAINT])
 
